extensions/target-specific/chromium/win-d3d11-angletextures/AngleTextures.js

```diff
@@ -0,0 +1,99 @@
+//--------------------------------------------------------------
+//
+//    MIT License
+//
+//    Copyright (c) Microsoft Corporation. All rights reserved.
+//
+//--------------------------------------------------------------
+
+"use strict";
+
+var AngleTextures = undefined;
+Loader.OnLoad(function () {
+  AngleTextures = {
+    GetRoots: function () {
+      return DbgObject.global("libGLESv2", "gSingleThreadedContext").catch((error)=>{
+        var errorMessage = ErrorMessages.CreateErrorsList(error) +
+                    ErrorMessages.CreateErrorReasonsList("Extension works in windows only and when Angle uses D3D11 backend.", ErrorMessages.SymbolsUnavailable);
```

Comment:
Going forward, we should find a more declarative way of handling platform-specific extensions. Ideally, this extension wouldn't even show up on non-Windows platform. Perhaps we need to add something in extension.json. Can you file an issue for this?

Reply:
I dont think this is tenable, for this extension you would need to know if Chrome is using SwiftShader or D3D11backend which you cannot tell unless you break in on the other hand if this is a general concept then If JsDBG intends to be untied from Chromium/Edge I don't think marking extensions as per platform would be important.

Reply:
Ideally, Windows-specific extensions shouldn't even be loaded/served on Mac/Linux (and vice versa). It's not just this one, we also have TEB and may have more in the future. They just add clutter on other platforms, so it's best to hide them from the user.

Reply:
We do something similar to hide the mshtml/edgehtml extensions. Seeing those in your view while debugging Chromium would be annoying. :)